src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```

Comment:
Variable name is too generic. Please raname it so that it reflects what kind of values it generates.